server/index.js

```diff
@@ -1,5 +1,5 @@
 const http = require('http');
-const fs   = require('fs');
+const fs = require('fs');
```

Comment:
Lose align
```js
const http = require('http');
const fs   = require('fs');
```

Reply:
strange, I have it perfectly aligned
![image](https://user-images.githubusercontent.com/49588154/92111333-d33cc600-edf4-11ea-91ef-0b6301e0cd0a.png)

Reply:
edited